pkg/kubectl/cmd/plugin/plugin.go

```diff
@@ -230,7 +226,10 @@ func isExecutable(fullPath string) (bool, error) {
 	}
 
 	if runtime.GOOS == "windows" {
-		if strings.HasSuffix(info.Name(), ".exe") {
+		fileExt := strings.ToLower(filepath.Ext(fullPath))
+
+		switch fileExt {
+		case ".bat", ".cmd", ".com", ".exe":
```

Comment:
What about the following file extensions ?

lnk -> File Shortcut
bin -> Binary Executable
ps1 -> Windows PowerShell Cmdlet
sct -> Windows Scriptlet
vb , vbs -> VBScript File

If some of the above extensions should be added, I can send out a PR.

Thanks

Reply:
i don't think some of these will integrate well with the plugin idea.
i'm +1 for ps1

> lnk -> File Shortcut

link files are mostly about GUI shortcuts

> bin -> Binary Executable

it's a binary file of a generic type. different tools have different interpretation of the contents.

> ps1 -> Windows PowerShell Cmdlet

this one is nice to have.

> sct -> Windows Scriptlet
> vb , vbs -> VBScript File

i think we can add these only if users demand them.

Reply:
> i think we can add these only if users demand them.

I'd be in favor of having this be the criteria overall for adding any additional extensions